src/providers/comments-provider.tsx

```diff
@@ -19,10 +20,14 @@ export const CommentsProvider: React.VoidFunctionComponent<{
 }> = ({ children }) => {
   const [comments, setComments] = React.useState<Comment[]>([]);
   const [newCommentText, setNewCommentText] = React.useState<string>("");
+  const { accidentId } = useAccident();
 
   React.useEffect(() => {
     const initFetchComments = async () => {
-      const initComments = await fetchComments(window.location.origin);
+      const initComments = await fetchComments(
+        window.location.origin,
+        accidentId,
+      );
       setComments(initComments);
```

Comment:
```suggestion
      const initialComments = await fetchComments(
        window.location.origin,
        accidentId,
      );
      setComments(initialComments);
```

(`initComments` looks like it is something callable)